# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/events/EventListenerManager.cpp
# Commit: 9355004c7f6a
# Full Hash: 9355004c7f6a72ca7f8c11df9d70076626767ae9
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 09:08:22
# Regressor Bug: 1742437
# File Overlap Count: 1
# Description:
#   Bug 1742437 - move mWebExtGlobal from ScriptLoadRequest to DOMScriptLoadContext; r=jonco,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D136744
# ==============================================================================

diff -r 0a5f61274aed -r 9355004c7f6a dom/events/EventListenerManager.cpp
--- a/dom/events/EventListenerManager.cpp	Thu Feb 17 15:57:48 2022 +0000
+++ b/dom/events/EventListenerManager.cpp	Thu Feb 17 15:57:49 2022 +0000
@@ -1236,7 +1236,7 @@
 
   RefPtr<ScriptFetchOptions> fetchOptions = new ScriptFetchOptions(
       CORS_NONE, aElement->OwnerDoc()->GetReferrerPolicy(),
-      aElement->OwnerDoc()->NodePrincipal(), nullptr);
+      aElement->OwnerDoc()->NodePrincipal());
 
   RefPtr<EventScript> eventScript =
       new EventScript(fetchOptions, uri, aElement);